fastlane/lib/fastlane/actions/github_api.rb

```diff
@@ -89,12 +89,22 @@ def available_options
             FastlaneCore::ConfigItem.new(key: :api_token,
                                          env_name: "FL_GITHUB_API_TOKEN",
                                          description: "Personal API Token for GitHub - generate one at https://github.com/settings/tokens",
+                                         conflicting_options: [:api_bearer],
                                          sensitive: true,
                                          code_gen_sensitive: true,
                                          is_string: true,
                                          default_value: ENV["GITHUB_API_TOKEN"],
                                          default_value_dynamic: true,
-                                         optional: false),
+                                         optional: true),
+            FastlaneCore::ConfigItem.new(key: :api_bearer,
+                                         env_name: "FL_GITHUB_API_BEARER",
+                                         sensitive: true,
+                                         code_gen_sensitive: true,
+                                         is_string: true,
+                                         description: "Use a Bearer authorization token (Usually generated by Github Apps). This will override api_token",
```

Comment:
```suggestion
                                         description: "Use a Bearer authorization token. Usually generated by Github Apps, e.g. GitHub Actions GITHUB_TOKEN environment variable",
```

Not sure if it's worth mentioning that "This will override api_token" because you added this parameter as a conflicting option `api_token`, so if the user provides both, it will fail early.